template.adoc

```diff
@@ -33,7 +33,7 @@
 = Title of guide, starting with a gerund (Creating, Administering, and so on)
 
 // Start the introduction with "You'll explore how to..." or something similarly catchy:
-You'll explore how to....
+Learn how to....
 // Write up to two sentences and fifty words, with meaningful information on what the user can accomplish with this guide.

```

Comment:
An ellipsis is a set of three periods ( . . . ) indicating an omission.  Looks like we have an extra one.